flags/flags.go

```diff
@@ -0,0 +1,279 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Stringer is only implemented by types which do not contain bit flags nor bitmasks,
+// as those can't be autogenerated by stringer.
+// TODO(#4): implement a custom stringer tool that understands bit flags and bitmasks.
+//go:generate stringer -type=ElementType -trimprefix ElementType_
+//go:generate stringer -type=AssemblyHashAlgorithm -trimprefix AssemblyHash_Algorithm
+
+package flags
+
+// AssemblyHashAlgorithm is defined in §II.23.1.1.
+type AssemblyHashAlgorithm uint32
+
+const (
+	AssemblyHash_AlgorithmNone AssemblyHashAlgorithm = 0x0000
+	AssemblyHash_AlgorithmMD5  AssemblyHashAlgorithm = 0x8003
+	AssemblyHash_AlgorithmSHA1 AssemblyHashAlgorithm = 0x8004

```

Comment:
When I saw `-trimprefix ElementType_` vs. `-trimprefix AssemblyHash_Algorithm` in the stringer gen lines I noticed this `_` seems to be in the wrong place:

```suggestion
	AssemblyHashAlgorithm_None AssemblyHashAlgorithm = 0x0000
	AssemblyHashAlgorithm_MD5  AssemblyHashAlgorithm = 0x8003
	AssemblyHashAlgorithm_SHA1 AssemblyHashAlgorithm = 0x8004
```